core/frontend/src/tile/map/ImageryProviders/WmsMapLayerImageryProvider.ts

```diff
@@ -100,17 +115,59 @@ export class WmsMapLayerImageryProvider extends MapLayerImageryProvider {
 
   private getVisibleQueryableLayersString(): string {
     const layers = new Array<string>();
-    const queryables = this.getQueryableLayers();
-    const visibles = this.getVisibleLayers();
-    queryables.forEach((layer: string) => { if (visibles.includes(layer)) layers.push(layer); });
+    const queryable = this.getQueryableLayers();
+    const visibleLayerNames = this.getVisibleLayers().map((layer) => layer.name);
+    queryable.forEach((layer: string) => { if (visibleLayerNames.includes(layer)) layers.push(layer); });
     return layers.join("%2C");
   }
 
+  public getCrsSupport(): {support3857: boolean,  support4326: boolean} {
```

Comment:
Why not make `WmsCrsSupport` public instead of internal and have this function return that type?

Reply:
Absolutely!